### ISSUE TITLE
missing processes for logical operations

### ISSUE THREAD

**Issue by clausmichele** [Open-EO/openeo-openshift-driver#125]
In EURAC use case, we use logical operators such as if, and, or, not, lt, gt and so on.
Do you plan to implement them? How much effort would it be for you?

**Comment by lforesta** [comment]
They are already all implemented here: [https://github.com/Open-EO/openeo-processes-python/blob/master/src/openeo_processes/logic.py](https://github.com/Open-EO/openeo-processes-python/blob/master/src/openeo_processes/logic.py)

You can find examples on how to use them in the tests: [https://github.com/Open-EO/openeo-processes-python/blob/master/tests/test_logic.py](https://github.com/Open-EO/openeo-processes-python/blob/master/tests/test_logic.py)

**Comment by clausmichele** [comment]
I'm currently using the web editor and in the EODC instance I don't see them, is it a client specific problem?

**Comment by lforesta** [comment]
No, those processes are not yet deployed, so they're not visible and not yet usable with our backend. This should be fixed this week hopefully.

But the processes themselves are implemented already, you will need to use them with you backend, so this is independent from the original issue you posted.

**Comment by clausmichele** [comment]
Ok, but I've opened this issue since we would need these processes to run our use case on EODC backend.
Maybe I misunderstood the difference between EODC implementation of the backend accessible with the OpenEO clients and the repo of the driver itself.
